configuration.yaml

```diff
@@ -0,0 +1,490 @@
+    type: string
+    default_value: ""
+  TF_CPP_VMODULE:
+    description:
+      - Environment variable used for TF VLOGs and takes the form of
+        TF_CPP_VMODULE=name=value,.... Note that for VLOGs you must set
+        TF_CPP_MIN_LOG_LEVEL=0. For PyTorch/XLA using a configuration like
+        TF_CPP_VMODULE=tensor=5 would enable logging.
+    type: string
+  TF_CPP_MIN_LOG_LEVEL:
+    description:
+      - Level to print messages for. TF_CPP_MIN_LOG_LEVEL=0 will turn on INFO
+        logging, TF_CPP_MIN_LOG_LEVEL=1 WARNING and so on. Our PyTorch/XLA
+        TF_VLOG uses tensorflow::INFO level by default so to see VLOGs set
+        TF_CPP_MIN_LOG_LEVEL=0. 
+    type: int
+    default_value: 1
+  TORCH_TEST_DEVICES:
+    description:
+      - Provided by the upstream and used to test new device types.
+    type: string
+  XLA_FNTRACKER_FILE:
+    description:
+      - If set, the path to a file where output from the function tracker
+        should be written.
+    type: string
+    default_value: ""
+  XLA_FNTRACKER_LIST:
+    description:
+      - Tags for the tracker context, which tell the function tracker which
+        functions to track.
+    type: string
+    default_value: ""
+  XLA_LAYOUTS:
+    description:
+      - A list of key/value pairs of the format "k1=v1;k2=v2". Keys are Shapes
+        and values are layouts.
+    type: string
+    default_value: ""
+  XLA_RNG_BIT_GENERATOR:
+    description:
+      - String name of the bit generator type, which can be either default,
+        philox, or three_fry. No default value because in that case there's
+        special behavior.
+    type: string
+  XLA_SLOW_COMPILE_HLO_FOLDER:
+    description:
+      - Folder name to save HLO files to, when the compile time is above a
+        certain threshold.
+    type: string
+    default_value: ""
+  XLA_TEST_DUMP_GRAPHS:
+    description:
+      - Type of graph to print to std::cerr. It can be empty, text, hlo, or
+        dot.
+    type: string
+    default_value: ""
+  PT_XLA_DEBUG_FILE:
+    description:
+      - If set, filepath used for printing out reports.
+    type: string
+    default_value: ""
+  TORCH_XLA_VERSION:
+    description:
+      - Specifies the version of PyTorch/XLA, rather than the hard-coded
+        version in setup.py; used when we're building binaries for
+        distribution. Should be parseable as a version number, e.g. 1.14
+    type: string
+    default_value: "1.14"
+  TORCH_XLA_PACKAGE_NAME:
+    description:
+      - Allows the developer to change the package name to something other than
+        torch_xla.
+    type: string
+    default_value: "torch_xla"
+  XLA_METRICS_PERCENTILES:
+    description:
+      - List of metrics percentiles to record.
+    type: string
+    default_value: "0.01:0.05:0.1:0.2:0.5:0.8:0.9:0.95:0.99"
+  GRPC_VERBOSITY:
+    description:
+      - Verbosity level for GRPC, e.g. INFO, ERROR, etc.
+    type: string
+    default_value: "ERROR"
+  TPU_ML_PLATFORM:
```

Comment:
lol I felt like we should not make this configurable

Reply:
I wonder if we should include it here for the sake of documentation, and remark that it shouldn't be changed. My intention in including it was just to cover every environment variable that shows up in this repo.
https://github.com/pytorch/xla/blob/f8581559272003d8828f15e2207e794e462ad463/torch_xla/__init__.py#L66